core/src/main/java/org/eclipse/hono/tracing/TracingHelper.java

```diff
@@ -159,6 +159,28 @@ public static void logError(final Span span, final String message) {
         }
     }
 
+    /**
+     * Marks an <em>OpenTracing</em> span as erroneous, logs a message and an error.
+     * <p>
+     * This method does <em>not</em> finish the span.
+     *
+     * @param span The span to mark.
+     * @param message The message to log on the span.
+     * @param error The error to log on the span.
+     * @throws NullPointerException if message or error is {@code null}.
```

Comment:
wouldn't it make more sense to allow any one of them to be `null` and then just log the non-`null` argument?

Reply:
:+1:  sounds good.